src/popup/components/Header.vue

```diff
@@ -93,12 +93,13 @@ export default defineComponent({
   },
   setup(props, { root }) {
     const isLoggedIn = useGetter('isLoggedIn');
+    const activeNetwork = useGetter('activeNetwork');
     const currentHomeRouteName = computed(() => (isLoggedIn.value) ? ROUTE_ACCOUNT : ROUTE_INDEX);
     const routeMeta = computed(() => root.$route.meta as WalletRouteMeta);
     const showHeaderNavigation = computed(() => !!routeMeta.value?.showHeaderNavigation);
     const isLogoDisabled = computed(() => root.$route.name === ROUTE_ACCOUNT);
     const titleTruncated = computed(
-      () => routeMeta.value?.title ? root.$t(`pages.titles.${routeMeta.value.title}`) : '',
+      () => routeMeta.value?.title ? root.$t(`pages.titles.${routeMeta.value.title}`) : activeNetwork.value?.name,
```

Comment:
I'm not sure about this logic. Now the network name will be displayed everywhere we don't have the title. But maybe the network name should only be visible on the Dashboard?